corehq/apps/integration/views.py

```diff
@@ -120,7 +120,16 @@ def gaen_otp_view(request, domain):
 
 
 def get_otp_response(post_data, gaen_otp_settings):
-    headers = {"Authorization": "Bearer %s" % gaen_otp_settings.auth_token}
+    headers = {}
+    if gaen_otp_settings.gaen_server_type == "Nearform":
+        headers = {"Authorization": "Bearer %s" % gaen_otp_settings.auth_token}
+
+    elif gaen_otp_settings.gaen_server_type == "APHL":
+        headers = {"x-api-key": "%s" % gaen_otp_settings.auth_token,
+                   "content-type": "application/json",
+                   "accept": "application/json"}
+        post_data = json.dumps(post_data)
```

Comment:
Do you definitely need to call `json.dumps`? I'd expect `requests` to known how to handle a dict when you're also passing the content type.

Reply:
when I don't have `json.dumps`, there is a 400 error "malformed json at position 2"

Reply:
Okay, cool, doesn't sound worth digging into more since it works.